bookwyrm/templates/preferences/edit_user.html

```diff
@@ -43,6 +43,12 @@
             {{ form.manually_approves_followers }}
         </label>
     </div>
+    <div class="block">
+        <label class="label" for="id_default_post_privacy">
+            Default post privacy:
+            {{form.default_post_privacy}}
```

Comment:
doing it this way means that instead of `private` the dropdown says `direct` because that's what's on the model. I was having trouble setting it up to use the privacy_select instead because I need to read more about this `form.save()` stuff, but my guess is that that's what's right here

Reply:
to use the `privacy_select.html` snippets, you'll need to have it take an optional parameter for the `name` of the form element (right now it's `<select name="privacy" id="privacy-{{ uuid }}">`), so that you can pass `default_post_privacy` in this instance. Otherwise the form will get the wrong name for that field and won't be able to find it on the model. I know my complex snippets setup is kind of a headache, so it's also completely fine to leave this as is, and I'll go in and modify the markup later.